ansible/environments.yml

```diff
@@ -103,3 +103,18 @@
           image_name: "d324da4b-b827-44fb-8331-bfad41a28aca"
       tags:
         - dev2-env
+
+    - name: Ensure aws_dev environment stack
+      cloudformation:
+        stack_name: "ae-aws-dev"
```

Comment:
Since we start using regions, better name will be `ae-uat-uswest2` (also change in the task name)